server/tests-py/queries/graphql_mutation/update/permissions/user_cannot_publish.yaml

```diff
@@ -0,0 +1,27 @@
+description: Try to change the published flag on an unpublished article
+url: /v1/graphql
+response:
+  errors:
+  - extensions:
+      path: $
+      code: permission-error
+    message: Check constraint violation. insert check constraint failed
```

Comment:
I think we'll need to change the message appropriately for update checks, `.. update check constraint failed`?